navbar-service.js

```diff
@@ -28,6 +32,23 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.site = {
+    title: config.data.siteTitle,
+    brand: config.data.style.brand,
+    url: '/'
+  };
```

Comment:
I think that using a route resolver seems like an end around solution. Setting the vars directly is straightforward, but I agree that there's some boilerplate there. If nothing else, we should at least do it the boilerplate way and then look for some way to reduce that in the future. IMO.